build/rpms/BUILD

```diff
@@ -0,0 +1,45 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@bazel_tools//tools/build_defs/pkg:rpm.bzl", "pkg_rpm")
+
+pkg_rpm(
+    name = "kubectl",
+    changelog = "CHANGELOG",
+    data = [
+        "//cmd/kubectl",
+    ],
+    spec_file = "kubectl.spec",
+    version = "1.6.0",
```

Comment:
Oh, I see. Well, `//:version` is just a genrule, so it'd be easy enough to create another file with the v stripped.

I agree that there should probably be more consistency around this; I'm not sure if anyone working on Bazel is looking at this holistically. Maybe we could start a thread on bazel-discuss?

The way I did it for `docker_bundle` (and for our own `gcs_upload`) worked well because we were already in python, so it was easy to add. Rules that don't fork out to python will be harder to update in the same way, at least not until Skylark can consume workspace info directly.

It's also confusing how some rules support make vars, but those are also in a completely different namespace, so they have to be escaped differently (at least for now).